src/setup.py

```diff
@@ -74,9 +74,12 @@
 except Exception as e:
     print(e)
     # 3.2
-    print("Can't find the game path, please enter the path manually: ")
-    gameFolder = input() # TODO: Check if the path is valid
-
+    CheckPathExists = False
+    while CheckPathExists == False: 
+        gameFolder = input("Can't find the game path, please enter the path manually: ")
+        if os.path.exists(gameFolder) == True and (os.path.exists(gameFolder+"hoi4.exe") == True or os.path.exists(gameFolder+"\\hoi4.exe") == True): 
```

Comment:
Here, you can do:

```python
CheckPathExists = os.path.exists(gameFolder)
```

I don't know if `os.path.exists(gameFolder+"\\hoi4.exe") ` is necessary.

Reply:
I'll tweak and do everything you wrote. "I don't know if os.path.exists(gameFolder+"\\hoi4.exe")  is necessary." It is obligatory because a person can write:"C:....\\Hearts of iron"  and then if + "hoi4.exe" -> "C:....\\Hearts of ironhoi4.exe"

Reply:
> I'll tweak and do everything you wrote. "I don't know if os.path.exists(gameFolder+"\hoi4.exe") is necessary." It is obligatory because a person can write:"C:....\Hearts of iron" and then if + "hoi4.exe" -> "C:....\Hearts of ironhoi4.exe"

Perfectly!